py/desispec/quicklook/qlconfig.py

```diff
@@ -123,17 +127,27 @@ def paargs(self,psfspfile=None):
                 paopts[PA]=paopt_skysub
             else:
                 paopts[PA]={}
+        #- Ignore intermediate dumping and write explicitly the outputfile for 
+        self.outputfile=self.dump_pa(self.palist[-1]) 
+
         return paopts 
         
-    def dump_pa(self,filetype):
+    def dump_pa(self,paname):
         """
         dump the PA outputs to respective files. This has to be updated for fframe and sframe files as QL anticipates for dumpintermediate case.
         """
+        pafilemap={'Preproc': 'pix', 'BoxcarExtract': 'frame', 'ApplyFiberFlat_QL': 'fframe', 'SkySub_QL': 'sframe'}
+        
+        if paname in pafilemap:
+            filetype=pafilemap[paname]
+        else:
+            raise IOError("PA name does not match any file type. Check PA name in config") 
+           
         if filetype in ["fframe","sframe"]: #- fiberflat fielded or sky subtracted intermediate files       
             pafile=os.path.join(self.specprod_dir,'exposures',self.night,"{:08d}".format(self.expid),"{}-{}-{:08d}.fits".format(filetype,self.camera,self.expid))
         else:
             pafile=findfile(filetype,night=self.night,expid=self.expid, camera=self.camera, rawdata_dir=self.rawdata_dir,specprod_dir=self.specprod_dir,outdir=self.outdir)
-            
+
         return pafile
 
     def dump_qa(self): 
```

Comment:
Same here also.  We had not wanted to add further instances flagrantly, but these are conventions based on those already existing and we've decided to stay with that.  We'll add relevant .fits and qa cases.